config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-trusted.yaml

```
@@ -96,16 +95,23 @@ periodics:
           exit 1
         fi
         echo "Running snyk scan .."
-        snyk test --json > ${ARTIFACTS}/result_unfiltered.json || if [ $? -gt 1 ]; then
-          echo "failed to run snyk scan"
+        RESULT_UNFILTERED=$(snyk test --json) || EXIT_CODE=$?
+        if [ $EXIT_CODE -gt 1 ]; then
```

```suggestion
        EXIT_CODE=0
        RESULT_UNFILTERED=$(snyk test --json) || EXIT_CODE=$?
        if [ $EXIT_CODE -gt 1 ]; then
```

If `snyk test --json` returns exit code 0 (less likely as it yields 1 for licenses/`0.0.0`), this will return `unbound variable` error for `EXIT_CODE` (note `set -u` at the top). 
Suggested to define `EXIT_CODE=0`